io-package.json

```diff
@@ -137,7 +137,7 @@
         },
         "tier": 3,
         "keywords": ["matter"],
-        "stopTimeout": 5000,
+        "stopTimeout": 15000,
```

Comment:
Realmente? Or was it for Testing?

Reply:
Really. In fact I increased to 30s in the end.
Depending how many devices you have online testing all that down and storing end state in db takes a bit. Normally 10s are enough.. but sometimes needs a bit longer.